apps/authentication-service/src/resources/clients/clientsService.ts

```diff
@@ -0,0 +1,87 @@
+import { Client, ClientCreate } from "auth0";
+
+import envs from "@/config";
+import { ICredentials } from "@/resources/clients/clientsModel";
+import { AuthService } from "@/shared/auth/authService";
+import { parseResponse } from "@/shared/http/httpService";
+
+const authDomain = envs.AUTH0_DOMAIN;
+const Auth = AuthService.getInstance();
+
+export const createClient = async (
```

Comment:
Create, delete, and getClient all have error handling if the request fails. We should have integration tests for that.